src/shared/ui/data-display/MainDetails/MainDetails.tsx

```diff
@@ -0,0 +1,46 @@
+import { Image, Link } from "@/shared/ui";
+
+interface MainDetailsProps {
+  artist: string;
+  picture: string;
+  pictureHeight: number;
+  pictureTopRounded?: boolean;
+  pictureWidth: number;
+  title: string;
+}
+
+/**
+ * Renders the main details of an album, including the artist, picture, picture dimensions, title, and picture styling.
+ */
+export function MainDetails({
+  artist,
+  picture,
+  pictureHeight,
+  pictureTopRounded,
+  pictureWidth,
+  title,
+}: MainDetailsProps) {
+  return (
+    <div className="flex items-center ">
+      <Image
+        alt="album cover"
```

Comment:
For now, can we rename it to `cover`? (as a reminder, the component will be used on several entities)

Reply:
Yes it is more generic with this, corrected.